spec/game_spec.rb

```diff
@@ -81,7 +81,7 @@
     allow(frame_double).to receive(:score).and_return(10, 3, 0)
     allow(frame_double).to receive(:spare?).and_return(true, false)
     allow(frame_double).to receive(:strike?).and_return(false)
-    allow(frame_double).to receive(:roll_1).and_return(3)
+    allow(frame_double).to receive(:roll_points).with(1).and_return(3)
```

Comment:
Wow the game spec looks interesting! - the way you used 'let' keywords - teach me how to! :)